components/dashboard/src/CookieSettingsModal.tsx

```diff
@@ -0,0 +1,108 @@
+            analytical: cookiePreferences.analytical,
+            targeting: cookiePreferences.targeting,
+        });
+
+        props.onSave();
+    };
+
+    const onClose = () => {
+        const value = getConsentCookies();
+        if (value) {
+            setCookiePreferences(JSON.parse(value));
+        }
+
+        props.onClose();
+    };
+
+    return (
+        <Modal visible={props.showCookieSettings} onClose={onClose}>
+            <h3 className="mb-4">Cookie settings</h3>
+            <div className="flex gap-2">
+                <h4>Strictly necessary cookies</h4>
+                <div className="-mt-4">
+                    <CheckBox title="" desc={undefined} checked={true} disabled={true} name="necessary" />
+                </div>
+            </div>
+            <p className="mb-4">
+                These are cookies that are required for the operation of our Site and under our terms with you. They
+                include, for example, cookies that enable you to log into secure areas of our Site or (on other sites)
+                use a shopping cart or make use of e-billing services.
+            </p>
+            <div className="flex gap-2">
+                <h4>Analytical / Performance cookies</h4>
+                <div className="-mt-4">
+                    <CheckBox
+                        title=""
+                        desc={undefined}
+                        checked={cookiePreferences.analytical}
+                        name="analytical"
+                        onChange={handleAnalyticalCheckboxChange}
+                    />
+                </div>
+            </div>
+            <p className="mb-4">
+                These allow us to recognise and count the number of visitors and to see how visitors move around our
+                site when they are using it. This helps us improve the way our Website works, for example, by ensuring
+                that users are finding what they are looking for easily.
+            </p>
+            <div className="flex gap-2">
+                <h4>Targeting/Advertising cookies</h4>
+                <div className="-mt-4">
+                    <CheckBox
+                        title=""
+                        desc={undefined}
+                        checked={cookiePreferences.targeting}
+                        name="targeting"
+                        onChange={handleTargetingCheckboxChange}
+                    />
+                </div>
+            </div>
+            <p className="mb-4">
+                These cookies record your visit to our Website, the pages you have visited and the links you have
+                followed. We will use this information subject to your choices and preferences to make our Website and
+                the advertising displayed on it more relevant to your interests. We may also share this information with
+                third parties for this purpose.
+            </p>
+            <button onClick={onSave}>Save</button>
```

Comment:
**suggestion**: In all other modals, we're currently using a border line as a separator between the modal content and the action buttons on the bottom.

- [ ] Could we also use this here?
- [ ] The sample applies also for the header section of this modal.